floris/floris_model.py

```diff
@@ -1722,3 +1722,72 @@ def reinitialize(self, **_):
             "The reinitialize method has been removed. Please use the set method. "
             "See https://nrel.github.io/floris/upgrade_guides/v3_to_v4.html for more information."
         )
+
+
+    @staticmethod
+    def merge_floris_models(fmodel_list, reference_wind_height=None):
+        """Merge a list of FlorisModel objects into a single FlorisModel object. Note that it uses
+        the very first object specified in fmodel_list to build upon,
+        so it uses those wake model parameters, air density, and so on.
+
+        Args:
+            fmodel_list (list): Array-like of FlorisModel objects.
+            reference_wind_height (float, optional): Height in meters
+                at which the reference wind speed is assigned. If None, will assume
+                this value is equal to the reference wind height specified in the FlorisModel
+                objects. This only works if all objects have the same value
+                for their reference_wind_height.
+
+        Returns:
+            fmodel_merged (FlorisModel): The merged FlorisModel object,
+                merged in the same order as fmodel_list. The objects are merged
+                on the turbine locations and turbine types, but not on the wake parameters
+                or general solver settings.
+        """
+
+        if not isinstance(fmodel_list[0], FlorisModel):
```

Comment:
@misi9170 @paulf81 I just came across this and I think it's not validating `fmodel_list` correctly. If I understand correctly that it's a 1-D list, then it's only validating the first element. You can check all elements with something like:
```python
# does `isinstance` on each element of the list through `map`
# then checks whether all the elements are True.
# If any elements are not True, then it goes into the if-statement
if not all( list( map(isinstance, fmodel_list, len(fmodel_list)*[FlorisModel]) ) ):

# or the same thing but more simply
if not all( type(fm) == FlorisModel for fm in fmodel_list ):
```

It may or may not be worth fixing right now, but I wanted to give you a heads up in case you see anything funny in the future with this function.